### PR TITLE
6848/uat feedback 23 download report format

### PR DESCRIPTION
**Pull request by mpetrovic**

### CODE REVIEW

File: career-planning-tool/index.html [hunk 1]
```diff
@@ -88,65 +88,89 @@ <h3>GS Level</h3>
...
-                                    {% endif %}
-                                {% endif %}
+                            {% for comp2 in general_comps %}
+                              {% if comp == comp2 %}
+                                {% assign sorted_competency_group_array = sorted_competency_group_array | push: comp | uniq %}
+                              {% endif %}
                             {% endfor %}
                         {% endfor %}
+                        
+                        {% for competency_group in job_comps %}
```
Comment (alexdntowe) [inline career-planning-tool/index.html:115]: please add comment for this block

File: career-planning-tool/index.html [hunk 2]
```diff
@@ -88,65 +88,89 @@ <h3>GS Level</h3>
...
+                                            <div class="career-input-group">
+                                              <label for="{{ competency_group_id | downcase }}" class="career-competency-input-groups-label text-underline text-accent-cool-darker display-inline-block font-sans-3xs"><strong class="cpt-strong">Select All</strong></label>
+                                                <input
+                                                    type="checkbox"
+                                                    id="{{ competency_group_id | downcase}}"
+                                                    data-filter="competency"
+                                                    data-major-group="job-specific"
+                                                    data-id="{{ competency_group_id | downcase }}"
+                                                    title="{{ competency_group }}"
+                                                    style="position: absolute; left: -9999px;"
```
Comment (alexdntowe) [inline career-planning-tool/index.html:134]: please us custom classes or USWDS utilities classes

File: career-planning-tool/index.html [hunk 3]
```diff
@@ -88,65 +88,89 @@ <h3>GS Level</h3>
...
+                                                />
+                                            </div>
+                                        </div>
+                                        </div>
+                                        {% for competency in competency_array %}
+                                        {% assign competency_id = competency | replace: ", ", "-" | replace: " ", "-" %}
+                                        <div class="career-competency-input-groups" style="display: none;">
```
Comment (alexdntowe) [inline career-planning-tool/index.html:141]: please us custom classes or USWDS utilities classes

File: career-planning-tool/index.html [hunk 4]
```diff
@@ -88,65 +88,89 @@ <h3>GS Level</h3>
...
+                            </div>
                         {% endfor %}
-
-
+                    </div>
+                    <div class="career-competency-level-1-input-group">
+                        <h3>General Career Competencies</h3>
+                        <div class="career-input-group">
+                            <label class="text-underline text-accent-cool-darker display-inline-block" for="general-career-competency-select-all"><strong>Select All</strong></label>
+                            <input style="position: absolute; left: -9999px;" type="checkbox" id="general-career-competency-select-all"/>
```
Comment (alexdntowe) [inline career-planning-tool/index.html:168]: please us custom classes or USWDS utilities classes

File: career-planning-tool/index.html (hunk 3 above)
Comment (mpetrovic) [reply career-planning-tool/index.html:141]: This style is temporary. It will be overridden based on user choices.

**Pull request merged**